src/index.html

```diff
@@ -17,6 +17,90 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="navigation_logo">
+          <a
+            class="logo_link"
+            href="#"
+          >
+            <img
+              class="logo"
+              alt="MOYO company logo"
+              src="images/logo.png"
+            />
+          </a>
+        </div>
```

Comment:
According to the checklist, you should add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. But don't add them between parent and child elements.